terraform/dev/eks-cluster.tf

```diff
@@ -0,0 +1,38 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.0.4"
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.24"
+
+  vpc_id                         = var.vpc_id
+  subnet_ids                     = ["subnet-a3f202fa", "subnet-fc843999", "subnet-0fc91a78"]
```

Comment:
Where do these come from?

Reply:
They're properties of our VPC (they basically correspond to different availability zones that our VPC is running in; we're here telling the cluster that it can operate in all these availability zones. We do have a fourth subnet but I didn't use it here because our existing deployments don't use it)